lib/modules/manager/kustomize/extract.ts

```diff
@@ -1,3 +1,4 @@
+import querystring from 'querystring';
```

Comment:
```suggestion
import querystring from 'node:querystring';
```

Should this have the `node:` prefix?

I'm also surprised to find that this would be our first use of this module.